app/models/gpttags.server.test.ts

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import { requestGpttags } from "../code/scout/gpttags";
+
+// Mocking axios.post method
+jest.mock("axios");
```

Comment:
we may have discussed.  vitest is already installed in the project and does mocking.  Is there a reason to use jest on top?